extensions/skyportal/static/js/components/ZTFAlertsByCoords.jsx

```diff
@@ -0,0 +1,361 @@
+
+  const userAccessibleGroups = useSelector(
+    (state) => state.groups.userAccessible
+  );
+
+  const userAccessibleGroupIds = useSelector((state) =>
+    state.groups.userAccessible?.map((a) => a.id)
+  );
+
+  const theme = useTheme();
+  const darkTheme = theme.palette.type === "dark";
+
+  const alerts = useSelector((state) => state.alertsByCoords);
+
+  const makeRow = (alert) => {
+    return {
+      objectId: alert?.objectId,
+      candid: alert?.candid,
+      jd: alert?.candidate.jd,
+      ra: alert?.candidate.ra,
+      dec: alert?.candidate.dec,
+      fid: alert?.candidate.fid,
+      mag: alert?.candidate.magpsf,
+      emag: alert?.candidate.sigmapsf,
+      programid: alert?.candidate.programid,
+    };
+  };
+
+  let rows = [];
+
+  if (alerts !== null && !isString(alerts) && Array.isArray(alerts)) {
+    rows = alerts.map((a) => makeRow(a));
+  }
+
+  useEffect(() => {
+    const fetchAlerts = async () => {
+      const data = await dispatch(Actions.fetchAlertsByCoords({ ra, dec, radius }));
+      setIsFetched(true);
+    };
+
+    if (!isFetched) {
+      fetchAlerts();
+    }
+  }, [dispatch, isFetched, ra, dec, radius]);
+
+  const classes = useStyles();
+
+  // This is just passed to MUI datatables options -- not meant to be instantiated directly.
+  const renderPullOutRow = (rowData, rowMeta) => {
+    const colSpan = rowData.length + 1;
+    const alertData = alerts[rowMeta.dataIndex];
+    const thumbnails = [
+      {
+        type: "new",
+        id: 0,
+        public_url: `/api/alerts/ztf/${alertData.objectId}/cutout?candid=${alertData.candid}&cutout=science&file_format=png`,
+      },
+      {
+        type: "ref",
+        id: 1,
+        public_url: `/api/alerts/ztf/${alertData.objectId}/cutout?candid=${alertData.candid}&cutout=template&file_format=png`,
+      },
+      {
+        type: "sub",
+        id: 2,
+        public_url: `/api/alerts/ztf/${alertData.objectId}/cutout?candid=${alertData.candid}&cutout=difference&file_format=png`,
+      },
+    ];
+
+    return (
+      <TableRow data-testid={`alertRow_${alertData.candid}`}>
+        <TableCell
+          style={{ paddingBottom: 0, paddingTop: 0 }}
+          colSpan={colSpan}
+        >
+          <Grid
+            container
+            direction="row"
+            spacing={3}
+            justify="center"
+            alignItems="center"
+          >
+            <Grid item>
+            <ThumbnailList
+              thumbnails={thumbnails}
+              ra={alertData.candidate.ra}
+              dec={alertData.candidate.dec}
+              displayTypes={["new", "ref", "sub"]}
+            />
+            </Grid>
+          </Grid>
+        </TableCell>
+      </TableRow>
+    );
+  };
+
+  const options = {
+    selectableRows: "none",
+    expandableRows: true,
+    renderExpandableRow: renderPullOutRow,
+    elevation: 1,
+    sortOrder: {
+      name: "jd",
+      direction: "desc",
+    },
+  };
+
+  const columns = [
```

Comment:
Can we also add ML score?

Reply:
What field is that? I don't see anything obvious here https://docs.fritz.science/catalogs.html#ztf-alerts

Reply:
drb

Reply:
is rb relevant too?

Reply:
and where are these fields documented?

Reply:
rb is not relevant. just drb. These fields are documented here 

https://zwickytransientfacility.github.io/ztf-avro-alert/schema.html

Reply:
then why is `rb` shown here along with `drb`? https://github.com/fritz-marshal/fritz/blob/master/extensions/skyportal/static/js/components/ZTFAlert.jsx#L221

Reply:
I've added both, as that's what Dima did on the above-linked alerts page

Reply:
rb is an older model that performs abysmally compared to drb and is no longer used in production